docs/fxa-engineering/release-process.md

```diff
@@ -65,6 +67,8 @@ After merging but before pushing, you should check the changelog to make sure th
 
 Then `git add` those changes and squash them into the preceding merge commit using `git commit --amend`. Now you can push and the merged changelog will make sense.
 
+### What happens if there are test conflicts?
```

Comment:
`test conflicts` or just `conflicts`?

Reply:
Good question. @vbudhram I grabbed this from a comment you'd written in Slack, I assume this is for conflicts and not test-specific conflicts?

Reply:
I better title might be

`What happens if there are merge conflicts (train-xxx => master)`

Merge conflicts in github PR is most likely the result of a recent patch to `master` branch. The easiest way to

1. Create a new branch from `train-xxx` call it `train-xxx-merge-master`
2. Merge the current `origin/master` into `train-xxx-merge-master`
3. Resolve any of the conflicts that occur in the merge
4. Commit changes and push to origin
5. Create a new PR `train-xxx-merge-master` -> `master`
6. Wait for Circle tests to pass and then merge when ready

You can close the PR with the conflict in it.